src/main/java/uk/ac/ebi/eva/pipeline/io/readers/VariantsMongoReader.java

```diff
@@ -69,7 +69,7 @@ public VariantsMongoReader(MongoOperations template, String collectionsVariantsN
         String[] fields = {"chr", "start", "end", "ref", "alt"};
```

Comment:
I can't comment on line 65 because it wasn't changed, but the field to query for the versioned annotation shouldn't be `annot.ct.so` but `annot.so`. The version should also be a mandatory argument.